features/jsonapi/jsonapi.feature

```diff
@@ -0,0 +1,206 @@
+Feature: JSON API basic support
+  In order to use the JSON API hypermedia format
+  As a client software developer
+  I need to be able to retrieve valid JSON API responses.
+
+  @createSchema
+  Scenario: Retrieve the API entrypoint
+    When I add "Accept" header equal to "application/vnd.api+json"
+    And I send a "GET" request to "/"
+    Then the response status code should be 200
+    And the response should be in JSON
+    And the header "Content-Type" should be equal to "application/vnd.api+json; charset=utf-8"
+    And the JSON node "links.self" should be equal to "/"
+    And the JSON node "links.dummy" should be equal to "/dummies"
+
+  Scenario: Test empty list against jsonapi-validator
+    When I add "Accept" header equal to "application/vnd.api+json"
+    And I send a "GET" request to "/dummies"
+    Then the response status code should be 200
+    And print last JSON response
+    And I validate it with jsonapi-validator
+    And the JSON node "data" should be an empty array
+
+  Scenario: Create a ThirdLevel
+    When I add "Content-Type" header equal to "application/vnd.api+json"
+    And I add "Accept" header equal to "application/vnd.api+json"
+    And I send a "POST" request to "/third_levels" with body:
+    """
+    {
+      "data": {
+        "type": "third-level",
+        "attributes": {
+          "level": 3
+        }
+      }
+    }
+    """
+    Then the response status code should be 201
+    # TODO: The response should have a Location header identifying the newly created resource
```

Comment:
Added to _Other desireble features_ on the PR description. I'll remove it from the code.